syllabus/syllabus-impl/src/java/org/sakaiproject/component/app/syllabus/SyllabusServiceImpl.java

```diff
@@ -1184,7 +1184,7 @@ public void transferCopyEntities(String fromContext, String toContext, List<Stri
 					SyllabusItem toSyItem = syllabusManager
 							.getSyllabusItemByContextId(toPage);
 					String redirectUrl = fromSyllabusItem.getRedirectURL();
-					if (redirectUrl.indexOf(fromContext) != -1)
+					if (redirectUrl != null && redirectUrl.contains(fromContext))
```

Comment:
Should use StringUtils.contains(redirectUrl, fromContext) here to eliminate null checks.